api/graphql/schema.graphql

```diff
@@ -28,38 +28,159 @@ enum Role {
+    isIndigenous: Boolean @rename(attribute: "is_indigenous")
+    isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
+
+    # Applicant info
+    jobLookingStatus: JobLookingStatus @rename(attribute: "job_looking_status")
     pools: [Pool] @hasMany # Pools that a user owns
     poolCandidates: [PoolCandidate] @hasMany # Pools candidates that a user owns
-    experiences: [Experience] # All experiences that a user owns
-    awardExperiences: [AwardExperience] # Award experiences that a user owns
-    communityExperiences: [CommunityExperience] # Community experiences that a user owns
-    educationExperiences: [EducationExperience] # Education experiences that a user owns
-    personalExperiences: [PersonalExperience] # Personal experiences that a user owns
-    workExperiences: [WorkExperience] # Work experiences that a user owns
+
+    hasDiploma: Boolean @rename(attribute: "has_diploma")
+    locationPreferences: [WorkRegion] @rename(attribute: "location_preferences")
+    locationExemptions: [String] @rename(attribute: "location_exemptions")
+    acceptedOperationalRequirements: [OperationalRequirement] @belongsToMany
+    expectedSalary: [SalaryRange] @rename(attribute: "expected_salary")
+    expectedClassifications: [Classification] @belongsToMany
+    wouldAcceptTemporary: Boolean @rename(attribute: "would_accept_temporary")
+    cmoAssets: [CmoAsset] @belongsToMany
+
+    # Experiences
+    experiences: [Experience] @hasMany # All experiences that a user owns
+    awardExperiences: [AwardExperience] @hasMany # Award experiences that a user owns
+    communityExperiences: [CommunityExperience] @hasMany # Community experiences that a user owns
+    educationExperiences: [EducationExperience] @hasMany # Education experiences that a user owns
+    personalExperiences: [PersonalExperience] @hasMany # Personal experiences that a user owns
+    workExperiences: [WorkExperience] @hasMany # Work experiences that a user owns
+}
+
+enum ProvinceOrTerritory {
+    BRITISH_COLUMBIA
+    ALBERTA
+    SASKATCHEWAN
+    MANITOBA
+    ONTARIO
+    QUEBEC
+    NEW_BRUNSWICK
+    NOVA_SCOTIA
+    PRINCE_EDWARD_ISLAND
+    NEWFOUNDLAND_AND_LABRADOR
+    YUKON
+    NORTHWEST_TERRITORIES
+    NUNAVUT
+}
+
+enum BilingualEvaluation {
+    COMPLETED_ENGLISH
+    COMPLETED_FRENCH
+    NOT_COMPLETED
+}
+
+enum EvaluatedLanguageAbility {
+    X
+    A
+    B
+    C
+    E
+    P
+}
+
+enum EstimatedLanguageAbility {
+    BEGINNER
+    INTERMEDIATE
+    ADVANCED
+}
+
+enum JobLookingStatus {
+    ACTIVELY_LOOKING
+    OPEN_TO_OPPORTUNITIES
+    INACTIVE
 }
 
 type Applicant {
     id: ID!
-    email: Email!
+    jobLookingStatus: JobLookingStatus @rename(attribute: "job_looking_status")
+
+    # Personal info
     firstName: String @rename(attribute: "first_name")
     lastName: String @rename(attribute: "last_name")
+    email: Email!
     telephone: PhoneNumber
+    currentProvince: ProvinceOrTerritory @rename(attribute: "current_province")
```

Comment:
Elsewhere we use custom defined "Work Regions" instead of provinces - can we double-check with design that we don't want to stick with one or the other?

Reply:
@Jerryescandon Would we prefer to use actual provinces or just Work Regions for a users current province? The wireframe doesn't specify.